jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/ee/rs/formparam/sub/JAXRSSubClientIT.java

```diff
@@ -52,22 +56,33 @@
 public class JAXRSSubClientIT extends JAXRSClientIT {
 
   public JAXRSSubClientIT() {
-    setup();
     setContextRoot("/jaxrs_ee_formparam_sub_web/resource/sub");
   }
 
-  private static final long serialVersionUID = 1L;
+  @ArquillianResource
+  @OperateOnDeployment("jaxrs_ee_formparam_sub_deployment")
+  private URL url;
 
   @BeforeEach
-  void logStartTest(TestInfo testInfo) {
-    TestUtil.logMsg("STARTING TEST : "+testInfo.getDisplayName());
-  }
+  public void setup() {
+
+    TestUtil.logTrace("setup method JAXRSSubClientIT");
+
+    assertFalse((url==null), "[JAXRSSubClientIT] 'url' was not injected.");
+    
+    String hostname = url.getHost();
+    String portnum = Integer.toString(url.getPort());
+    
+    assertFalse(isNullOrEmpty(hostname), "[JAXRSSubClientIT] 'webServerHost' was not set.");
+    _hostname = hostname.trim();
+    assertFalse(isNullOrEmpty(portnum), "[JAXRSSubClientIT] 'webServerPort' was not set.");
+    _port = Integer.parseInt(portnum.trim());
+    TestUtil.logMsg("[JAXRSSubClientIT] Test setup OK");
 
-  @AfterEach
-  void logFinishTest(TestInfo testInfo) {
-    TestUtil.logMsg("FINISHED TEST : "+testInfo.getDisplayName());
   }
 
+  private static final long serialVersionUID = 1L;
+
```

Comment:
This shouldn't be required.

Reply:
Agree. It was not introduced in this PR, diff has the same line removed at the beginning. Hopefully these along with other occurrences can be addressed in later changes.